CONTRIBUTING.md

```diff
@@ -43,6 +43,60 @@ Use these commands whenever you want to work on one of these packages. Ideally,
 | `pnpm header:start`               | starts the stencil compiler for the header                                                   |
 | `pnpm icons:start`                | starts the http server for debugging downloaded icons                                        |
 
+### Definition of Done for a Story
+
+#### Component docs
+
+- Short introduction
+- Canvas for showcasing
+- Controls for the showcase
+- Variants if applicable (e.g. group of component or small example in context)
+- Instructions for installation
+
+#### Controls
+
+- Possibility to showcase and test all variants of the component. This enables re-usage of the docs programmatically for visual regression testing.
+- Accessibility hints for relevant controls like disabled state or hidden labels
+
+#### Testing
+
+- Cross browser testing of the docs (official Post supported browsers -> https://github.com/swisspost/design-system/blob/main/packages/styles/.browserslistrc)
```

Comment:
Maybe this is something to be reviewed since: https://github.com/swisspost/design-system/issues/2425